scientific_library/tvb/tests/library/datatypes/surfaces_test.py

```diff
@@ -117,7 +120,9 @@ def test_cortical_topology_isolated_vertex(self):
         dt = surfaces.Surface()
         dt.vertices = numpy.array([[0, 0, 0], [1, 0, 0], [0, 1, 0], [0, 0, 1], [0, 0, 2]]).astype(numpy.float64)
         dt.triangles = numpy.array([[0, 2, 1], [0, 1, 3], [0, 3, 2], [1, 2, 3]])
-        dt.configure()
+
+        with numpy.errstate(all='ignore'):
+            dt.configure()
```

Comment:
Here, also, I see two dummy surfaces define above. Would there be a simple way to fix them (e.g. make sure it is a cube) ?